crates/sui-core/src/unit_tests/authority_tests.rs

```diff
@@ -1916,36 +1916,6 @@ async fn test_type_argument_dependencies() {
     ));
 }
 
-#[tokio::test]
```

Comment:
I am really not clear what this test was checking.
I have update the order in which input object checks and gas checks are performed and this test crashes (panic).
It was previously failing "gracefully" over a gas check (coin owned by sender).
However if I make the coin correctly owned by the sender, the test still crashes because it is truly sending a "transaction" that breaks invariants (only execution and no signing, where signing would have failed correctly).
So I have deleted it and hope someone can tell me what to do